adapter/memquota2/dedup.go

```diff
@@ -105,7 +105,7 @@ func (du *dedupUtil) reapDedup() {
 	du.oldDedup = du.recentDedup
 	du.recentDedup = t
 
-	if du.logger.VerbosityLevel(4) {
+	if len(t) > 0 && du.logger.VerbosityLevel(4) {
```

Comment:
we have at least 3 PRs with 3 different fixes for this.  let's settle on one approach.

Reply:
I will remove this file from my PR, should not have checked it in. It just locally annoying me.

Reply:
yeah, it is super annoying. i'm fine with your fix... or zack's making it level 6... or which ever... it just needs to go in.